install.sh

```diff
@@ -4049,8 +616,9 @@ setup_hook_templates() {
 ############################################################
 install_command_line_tool() {
     mkdir -p "$INSTALL_DIR/bin" &&
-        echo "$CLI_TOOL_CONTENT" >"$INSTALL_DIR/bin/githooks" &&
-        chmod +x "$INSTALL_DIR/bin/githooks" &&
+        echo "#!/bin/sh" >"$INSTALL_DIR/bin/githooks" &&
+        echo "sh '$INSTALL_DIR/release/cli.sh' \"\$@\"" >>"$INSTALL_DIR/bin/githooks"
+    chmod +x "$INSTALL_DIR/bin/githooks" &&
```

Comment:
The `chmod` should be chained with `&&` so we only do it if previous commands were successful, I'll fix it. /after